wgpu/src/backend/direct.rs

```diff
@@ -41,7 +41,7 @@ impl fmt::Debug for Context {
 }
 
 impl Context {
-    #[cfg(any(not(target_arch = "wasm32"), feature = "emscripten"))]
+    #[cfg(any(not(target_arch = "wasm32"), target_os = "emscripten"))]
```

Comment:
Can be removed since it's now a subset of the guard on the module.
There are a bunch more instances of this in this file.

Reply:
I believe these are correct, because they the guard on the module allows for `webgl`, but this guard here doesn't.

Reply:
These _should_ allow for `webgl` - as webgl uses wgpu-hal

Reply:
Alright, done!

Reply:
Note that this does apply to all the guards in the file besides the ones that use web_sys

Reply:
This was definitely not enabled under `webgl` before, and I have to adjust the guards in `Instance` too for that.
Could we do this in a different PR? I'm happy to open one right after to address this.

Reply:
Cool, that's fine.